etc/containerpilot.json5

```diff
@@ -0,0 +1,34 @@
+{
+  consul: "127.0.0.1:8500",
+  jobs: [
+    {
+      name: "preStart",
+      exec: ["/usr/local/bin/consul-manage", "preStart"],
+    },
+    {
+      name: "consul",
+      port: 8500,
+      exec: ["/bin/consul", "agent", "-server",
+             "-bootstrap-expect", "3",
```

Comment:
When I went to go do the env vars I realized we _do_ have a 3-node demo in the `local-compose.yml`. So really we're just talking about reusing this instead of `consul:latest` for being consumed by other blueprints as a one-node. For demos we _really_ want to use `-dev` mode for Consul rather than `--bootstrap-expect=1` (for time-to-launch purposes), so I'll try to crowbar that into the env var setup here somehow.